images/php/fpm/Dockerfile

```diff
@@ -41,10 +41,11 @@ COPY ssmtp.conf /etc/ssmtp/ssmtp.conf
 # @see https://docs.newrelic.com/docs/agents/php-agent/getting-started/php-agent-compatibility-requirements
 ENV NEWRELIC_VERSION=9.10.1.263
 
-# Pin curl to Version 7.69.1-r0 as the current shipped one 7.67.0 has a bug, see
-# https://github.com/curl/curl/issues/4624
+# Pin curl to Version 7.70.0-r0 as the current shipped one 7.67.0 has a bug.
+# @see https://github.com/curl/curl/issues/4624
+# @see https://pkgs.alpinelinux.org/packages?name=curl&branch=v3.11
 # TODO: Remove as soon as Alpine 3.11 is shipped with a version higher than 7.67.0
-RUN apk add --no-cache curl=7.69.1-r0 libcurl=7.69.1-r0 --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
+RUN apk add --no-cache curl=7.70.0-r0 libcurl=7.70.0-r0 --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
```

Comment:
The version doesn't actually need to be defined here. It will just pull in the latest version in `edge`. Also `libcurl` is a `curl` dependency, so this can be simplified:
```
apk --no-cache add curl --repository http://dl-cdn.alpinelinux.org/alpine/edge/main/
```

Reply:
Nice catch.